third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+**Trigger** - A new `Intel RealSense` device is connected locally (e.g, USB) and detected by the **server**.
+
+**Successful Result** - The **user** will receive all needed data to interact with the device
+
+**Main Scenario**
+1. The **server** will create a new *device-info* writer to publish this connected device existence.
+2. Subscribed **user**s are matched for this writer.
+3. The **server** publishes *device-info* message with this device details. The *device-info* contains a *topic-root* field. All communication to and from the camera is done using topics containing this *topic-root* in their names (see [ICD](https://github.com/IntelRealSense/librealsense/blob/dds/doc/DDS/DDS%20ICD.md)).
+4. The **user** subscibes to the appropriate *notification* topic using the *topic-root* field (e.g., `realsense/D435I/112322073677/notification`).
+5. The **server** will recognize the subscription and will send discovery *notification* messages in this order:
+    1. A *device-header* message
+    2. A *device-options* message if there are device level options
+    3. for each supported stream
+        - A *stream-header* message
+        - A *stream-options* message
+
+**Variations**
+1. A new **user** joins the network, subscribing to the *device-info* topic.
+    1. Continue from step 2 of the Main Scenario.
+
+**Exceptions**
+1.
+
+**Constraints**
+1. Discovery *notification* messages timeout is 30 seconds.
```

Comment:
It is HUGE! Maybe we should warn the users in bigger font or something like that

Reply:
ok